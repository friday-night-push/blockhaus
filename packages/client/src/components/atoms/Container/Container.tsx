import { Flex, FlexProps } from '@gravity-ui/uikit';

export type ContainerProps = FlexProps;

export const Container = ({ children, ...props }: ContainerProps) => {
  return (
    <section>
      <Flex gap="3" {...props}>
        {children}
      </Flex>
    </section>
  );
};
