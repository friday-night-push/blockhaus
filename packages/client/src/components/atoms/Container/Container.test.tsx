import { render } from '@testing-library/react';

import { Container } from './Container';

describe('Container', () => {
  it('renders correctly with default props', () => {
    const { container } = render(<Container></Container>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
