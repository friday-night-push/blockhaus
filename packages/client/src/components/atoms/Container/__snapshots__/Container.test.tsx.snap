// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly with default props 1`] = `
<section
  class="g-box g-flex"
  style="column-gap: 12px; row-gap: 12px;"
/>
`;
